htdocs/core/lib/security.lib.php

```diff
@@ -245,7 +245,7 @@ function dolDecrypt($chain, $key = '')
  * 										Use 'md5' if hash is not needed for security purpose. For security need, prefer 'auto'.
  * 	@param 		int 		$nosalt		Do not include any salt
  *  @param		int			$mode		0=Return encoded password, 1=Return array with encoding password + encoding algorithm
- * 	@return		string|array<pass_encrypted:string,pass_encoding:string>			Hash of string or array with pass_encrypted and pass_encoding
+ * 	@return		string|array(pass_encrypted:string,pass_encoding:string)			Hash of string or array with pass_encrypted and pass_encoding
```

Comment:
Why changing this ?
Before the change, the syntax was correct.

Reply:
sorry that's a try to be clean against phpcs/phan/other rules who seems to refuse that PR due to line 248 syntax ...

Reply:
![image](https://github.com/user-attachments/assets/290d231b-8551-46d2-96e4-78091c1467f3)

Reply:
@rycks merge the branch develop, i think is fixed